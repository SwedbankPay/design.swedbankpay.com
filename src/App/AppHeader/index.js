import React from "react";
import { withRouter } from "react-router-dom";

import { SelectPanel } from "../utils";

import routes from "../routes/all";

const basename = process.env.basename;
const brand = process.env.brand;

const AppHeader = () => (
    <header id="dg-topbar" className="topbar topbar-max-md-sticky d-md-flex d-lg-none designguide-header">
        <button type="button" className="topbar-btn">
            <i className="material-icons topbar-btn-icon" aria-hidden="true">menu</i>
        </button>
        <button type="button" className="topbar-close">
            <i className="material-icons topbar-btn-icon" aria-hidden="true">close</i>
        </button>
        <a href="/" className="topbar-logo">{"\n"}
            <img src={`${basename}designguide/assets/${brand}-logo-h.svg`} alt={`${brand}-logo`} className="logotype-horizontal logotype-md"/> {"\n"}
        </a>
        <nav className="topbar-nav">
            <div className="topbar-link-container">
                <SelectPanel id="doc-topbar-sidebar" topbarId="dg-topbar" newSidebar={false} routes={routes} />
            </div>
        </nav>
    </header>
);

export default withRouter(AppHeader);
