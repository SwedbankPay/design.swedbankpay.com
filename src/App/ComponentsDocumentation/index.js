import React from "react";

import { RenderPage } from "../utils";

const ComponentsDocumentation = () => (
    <RenderPage path="/components" initPath="/components/components" />
);

export default ComponentsDocumentation;
