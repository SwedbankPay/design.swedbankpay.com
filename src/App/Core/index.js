import React from "react";

import { RenderPage } from "../utils";

const Core = () => (
    <RenderPage path="/core" initPath="/core/color" />
);

export default Core;
