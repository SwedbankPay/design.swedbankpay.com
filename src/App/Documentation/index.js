import React, { Suspense } from "react";
import { Switch, Route, Redirect } from "react-router-dom";

import { SelectPanel, RenderRoutes, LoadingComponent } from "../utils";
import routes from "../routes/docs";

const Documentation = () => (
    <div className="documentation">
        <div className="row">
            <div className="col-xxl-2 col-md-3 d-none d-md-block">
                <SelectPanel id="doc-siderbar" routes={routes} />
            </div>
            <main className="doc-view col-xxl-10 col-md-9">
                <Suspense fallback={<LoadingComponent />}>
                    <Switch>
                        <Route exact path="/docs" render={() => <Redirect to="/docs/getting-started" />} />
                        {routes.map(route => <RenderRoutes key={`renderRoutes_${route.title}`} {...route} appFolder={"Documentation"} />)}
                        <Redirect from="/docs/*" to="/404" />
                    </Switch>
                </Suspense>
            </main>
        </div>
    </div>
);

export default Documentation;
