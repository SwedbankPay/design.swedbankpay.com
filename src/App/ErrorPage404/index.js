import React from "react";

const ErrorPage404 = () => (
	<>
		<h1>This is not the page you are looking for.</h1>
	</>
);

export default ErrorPage404;
