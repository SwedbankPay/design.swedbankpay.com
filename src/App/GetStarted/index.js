import React from "react";

import { RenderPage } from "../utils";

const GetStarted = () => (
	<RenderPage path="/get-started" initPath="/get-started/introduction" />
);

export default GetStarted;
