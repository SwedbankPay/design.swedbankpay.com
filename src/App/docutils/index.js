import ComponentPreview from "./ComponentPreview";
import DocContainer from "./DocContainer";
import DocHeading from "./DocHeading";
import DocToc from "./DocToc";
import DeprecatedComponentAlert from "./DeprecatedComponentAlert";
import ExperimentalComponentAlert from "./ExperimentalComponentAlert";
import Icon from "./Icon";
import JavascriptDocs from "./JavascriptDocs";
import JavascriptDocElement from "./JavascriptDocElement";
import StatusBadge from "./StatusBadge";

export {
    ComponentPreview,
    DocContainer,
    DocHeading,
    DocToc,
    DeprecatedComponentAlert,
    ExperimentalComponentAlert,
    Icon,
    JavascriptDocs,
    JavascriptDocElement,
    StatusBadge
};
