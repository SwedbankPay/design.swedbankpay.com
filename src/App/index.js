import React, { Component, Suspense } from "react";
import { Router, Switch, Route, withRouter } from "react-router-dom";
import { createBrowserHistory } from "history";

import AppHeader from "./AppHeader";
import { LoadingComponent } from "./utils";
import SelectPanel from "./utils/SelectPanel";

import routes from "./routes/all";

const { topbar } = window.dg;

const BASENAME = process.env.basename || "/";

const history = createBrowserHistory({ basename: BASENAME });

class ScrollToTop extends Component {
    componentDidUpdate (prevProps) {
        if (this.props.location !== prevProps.location) {
            window.scrollTo(0, 0);
        }
    }

    render () {
        return this.props.children;
    }
}

const ScrollToTopComponent = withRouter(ScrollToTop);

const Home = React.lazy(() => import(/* webpackChunkName: "home.chunk" */ "./Home/index.js"));

const GettingStarted = React.lazy(() => import(/* webpackChunkName: "getting-started.chunk" */ "./GettingStarted/index.js"));

const Components = React.lazy(() => import(/* webpackChunkName: "components.chunk" */ "./ComponentsDocumentation/index.js"));

const Identity = React.lazy(() => import(/* webpackChunkName: "core.chunk" */ "./Identity/index.js"));

const Examples = React.lazy(() => import(/* webpackChunkName: "examples.chunk" */ "./Examples/index.js"));

const Utilities = React.lazy(() => import(/* webpackChunkName: "utilities.chunk" */ "./Utilities/index.js"));

const ErrorPage404 = React.lazy(() => import(/* webpackChunkName: "404.chunk" */ "./ErrorPage404/index.js"));

class App extends Component {
    constructor () {
        super();

        if (process.env.google) {
            history.listen(location => {
                window.gtag("config", "UA-3440932-20", {
                    /* eslint-disable camelcase */
                    page_location: window.location.href,
                    page_path: location.pathname
                    /* eslint-enable camelcase */
                });
            });
        }
    }

    componentDidMount () {
        topbar.init();
    }

    render () {
        return (
            <Router basename={BASENAME} history={history}>
                <ScrollToTopComponent>
                    <AppHeader />
                    <div className="documentation">
                        <div className="d-md-flex">
                            <div className="d-none d-lg-block">
                                <SelectPanel id="doc-sidebar" newSidebar={true} routes={routes} />
                            </div>
                            <main className="doc-view">
                                <Suspense fallback={<LoadingComponent />}>
                                    <Switch>
                                        <Route exact path="/" component={Home} />
                                        <Route path="/get-started" component={GettingStarted} />
                                        <Route path="/components" component={Components} />
                                        <Route path="/identity" component={Identity} />
                                        <Route path="/examples" component={Examples} />
                                        <Route path="/utilities" component={Utilities} />
                                        <Route path="/404" component={ErrorPage404} />
                                        <Route component={ErrorPage404} />
                                    </Switch>
                                </Suspense>
                            </main>
                        </div>
                    </div>
                </ScrollToTopComponent>
            </Router>
        );
    }
}

export default App;
