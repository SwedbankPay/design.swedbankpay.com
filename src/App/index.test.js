import React from "react";
import { shallow } from "enzyme";

import App from "./index";

describe("Main: App", () => {
    it("is defined", () => {
        expect(App).toBeDefined();
    });

    it("renders", () => {
        const wrapper = shallow(<App />);

        expect(wrapper).toMatchSnapshot();
    });
});
