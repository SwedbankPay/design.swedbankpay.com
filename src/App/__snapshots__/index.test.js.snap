// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main: App renders 1`] = `
<Router
  basename="/TEST/"
  history={
    Object {
      "action": "POP",
      "block": [Function],
      "createHref": [Function],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "length": 1,
      "listen": [Function],
      "location": Object {
        "hash": "",
        "pathname": "/",
        "search": "",
        "state": undefined,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
  <withRouter(ScrollToTop)>
    <withRouter(AppHeader) />
    <div
      className="documentation"
    >
      <div
        className="d-md-flex"
      >
        <SkipLink />
        <div
          className="d-none d-lg-block"
        >
          <withRouter(SelectPanel)
            id="doc-sidebar"
            newSidebar={true}
            routes={
              Array [
                Object {
                  "appFolder": "GetStarted",
                  "entryCardText": "Get started by setting up the Design Guide!",
                  "icon": Object {
                    "name": "wb_incandescent",
                    "rotate": true,
                  },
                  "path": "/get-started",
                  "redirect": "/get-started/introduction",
                  "routes": Array [
                    Object {
                      "componentPath": "get-started/Introduction",
                      "path": "/get-started/introduction",
                      "title": "Introduction",
                    },
                    Object {
                      "componentPath": "get-started/ForDevelopers",
                      "path": "/get-started/for-developers",
                      "title": "For developers",
                    },
                    Object {
                      "componentPath": "get-started/ForDesigners",
                      "path": "/get-started/for-designers",
                      "title": "For designers",
                    },
                  ],
                  "title": "Get started",
                  "wideCard": true,
                },
                Object {
                  "appFolder": "Identity",
                  "entryCardText": "Learn about our fundamental principles",
                  "icon": Object {
                    "name": "palette",
                  },
                  "path": "/identity",
                  "redirect": "/identity/identity",
                  "routes": Array [
                    Object {
                      "componentPath": "identity/Identity",
                      "path": "/identity/identity",
                      "title": "Overview",
                    },
                    Object {
                      "componentPath": "identity/Accessibility",
                      "path": "/identity/accessibility",
                      "text": "Principles & WCAG 2.1 guidelines",
                      "title": "Accessibility",
                    },
                    Object {
                      "componentPath": "identity/Color",
                      "path": "/identity/color",
                      "text": "Our color palette with HEX codes",
                      "title": "Color",
                    },
                    Object {
                      "componentPath": "identity/Copywriting",
                      "path": "/identity/copywriting",
                      "text": "Tonality & UX copy in our products",
                      "title": "Copywriting",
                    },
                    Object {
                      "componentPath": "identity/Grid",
                      "path": "/identity/grid",
                      "text": "Our grid and best practices",
                      "title": "Grid",
                    },
                    Object {
                      "componentPath": "identity/Iconography",
                      "path": "/identity/iconography",
                      "text": "Guidelines and icon library",
                      "title": "Iconography",
                    },
                    Object {
                      "componentPath": "identity/Imagery",
                      "path": "/identity/imagery",
                      "text": "Guidelines and image library",
                      "title": "Imagery",
                    },
                    Object {
                      "componentPath": "identity/Logotype",
                      "path": "/identity/logotype",
                      "statusBadges": Array [
                        "updated",
                      ],
                      "text": "The TESTBRAND logotype",
                      "title": "Logotype",
                    },
                    Object {
                      "componentPath": "identity/Page-layout",
                      "path": "/identity/page-layout",
                      "text": "See our layout structure",
                      "title": "Page layout",
                    },
                    Object {
                      "componentPath": "identity/Spacing",
                      "path": "/identity/spacing",
                      "text": "8-point spacing system",
                      "title": "Spacing",
                    },
                    Object {
                      "componentPath": "identity/Typography",
                      "path": "/identity/typography",
                      "statusBadges": Array [
                        "updated",
                      ],
                      "text": "See our fronts and sizing",
                      "title": "Typography",
                    },
                  ],
                  "title": "Identity",
                },
                Object {
                  "appFolder": "ComponentsDocumentation",
                  "entryCardText": "Find all currently available components here",
                  "icon": Object {
                    "name": "category",
                  },
                  "path": "/components",
                  "redirect": "/components/components",
                  "routes": Array [
                    Object {
                      "componentPath": "components/Components",
                      "path": "/components/components",
                      "title": "Overview",
                    },
                    Object {
                      "componentPath": "components/Accordion",
                      "icon": "calendar_view_day",
                      "outlined": true,
                      "path": "/components/accordion",
                      "statusBadges": Array [
                        "javascript",
                      ],
                      "title": "Accordion",
                    },
                    Object {
                      "componentPath": "components/ActionList",
                      "icon": "list",
                      "path": "/components/action-list",
                      "statusBadges": Array [
                        "javascript",
                      ],
                      "title": "Action list",
                    },
                    Object {
                      "componentPath": "components/Alerts",
                      "icon": "error_outline",
                      "path": "/components/alerts",
                      "statusBadges": Array [
                        "javascript",
                      ],
                      "title": "Alerts",
                    },
                    Object {
                      "componentPath": "components/Badge",
                      "icon": "more",
                      "outlined": true,
                      "path": "/components/badge",
                      "title": "Badges",
                    },
                    Object {
                      "componentPath": "components/Breadcrumb",
                      "icon": "last_page",
                      "outlined": true,
                      "path": "/components/breadcrumb",
                      "title": "Breadcrumb",
                    },
                    Object {
                      "componentPath": "components/Buttons",
                      "icon": "touch_app",
                      "outlined": true,
                      "path": "/components/buttons",
                      "title": "Buttons",
                    },
                    Object {
                      "componentPath": "components/Card",
                      "icon": "view_agenda",
                      "outlined": true,
                      "path": "/components/cards",
                      "title": "Cards",
                    },
                    Object {
                      "componentPath": "components/Charts",
                      "icon": "bar_chart",
                      "outlined": true,
                      "path": "/components/charts",
                      "statusBadges": Array [
                        "javascript",
                      ],
                      "title": "Charts",
                    },
                    Object {
                      "componentPath": "components/Checkbox",
                      "icon": "check_box",
                      "outlined": true,
                      "path": "/components/checkbox",
                      "statusBadges": Array [
                        "javascript",
                      ],
                      "title": "Checkbox",
                    },
                    Object {
                      "componentPath": "components/Datepickers",
                      "icon": "event",
                      "outlined": true,
                      "path": "/components/datepickers",
                      "statusBadges": Array [
                        "javascript",
                      ],
                      "title": "Datepickers",
                    },
                    Object {
                      "componentPath": "components/Dialog",
                      "icon": "picture_in_picture",
                      "outlined": true,
                      "path": "/components/dialog",
                      "statusBadges": Array [
                        "javascript",
                      ],
                      "title": "Dialog",
                    },
                    Object {
                      "componentPath": "components/Dropdown",
                      "icon": "search",
                      "outlined": true,
                      "path": "/components/dropdown",
                      "statusBadges": Array [
                        "javascript",
                        "new",
                      ],
                      "title": "Dropdown",
                    },
                    Object {
                      "componentPath": "components/Expandable",
                      "path": "/components/expandable",
                      "statusBadges": Array [
                        "javascript",
                      ],
                      "title": "Expandable",
                    },
                    Object {
                      "componentPath": "components/InputField",
                      "icon": "text_fields",
                      "path": "/components/input-field",
                      "statusBadges": Array [
                        "javascript",
                        "updated",
                      ],
                      "title": "Input field",
                    },
                    Object {
                      "componentPath": "components/Links",
                      "icon": "link",
                      "outlined": true,
                      "path": "/components/links",
                      "title": "Links",
                    },
                    Object {
                      "componentPath": "components/Lists",
                      "icon": "sort",
                      "outlined": true,
                      "path": "/components/lists",
                      "title": "Lists",
                    },
                    Object {
                      "componentPath": "components/Loader",
                      "icon": "loop",
                      "outlined": true,
                      "path": "/components/loader",
                      "title": "Loader",
                    },
                    Object {
                      "componentPath": "components/MediaObject",
                      "icon": "account_circle",
                      "outlined": true,
                      "path": "/components/media-object",
                      "title": "Media Object",
                    },
                    Object {
                      "componentPath": "components/Nav",
                      "icon": "explore",
                      "outlined": true,
                      "path": "/components/nav",
                      "statusBadges": Array [
                        "javascript",
                      ],
                      "title": "Navigation",
                    },
                    Object {
                      "componentPath": "components/Pagination",
                      "icon": "power_input",
                      "outlined": true,
                      "path": "/components/pagination",
                      "statusBadges": Array [
                        "javascript",
                      ],
                      "title": "Pagination",
                    },
                    Object {
                      "componentPath": "components/Panel",
                      "icon": "table_rows",
                      "outlined": true,
                      "path": "/components/panel",
                      "title": "Panel",
                    },
                    Object {
                      "componentPath": "components/ProgressSteps",
                      "icon": "hdr_strong",
                      "path": "/components/progress-steps",
                      "title": "Progress steps",
                    },
                    Object {
                      "componentPath": "components/RadioButton",
                      "icon": "radio_button_checked",
                      "path": "/components/radio-button",
                      "statusBadges": Array [
                        "javascript",
                      ],
                      "title": "Radio button",
                    },
                    Object {
                      "componentPath": "components/Rangeslider",
                      "icon": "tune",
                      "outlined": true,
                      "path": "/components/rangeslider",
                      "statusBadges": Array [
                        "javascript",
                      ],
                      "title": "Rangeslider",
                    },
                    Object {
                      "componentPath": "components/Sheet",
                      "icon": "vertical_split",
                      "outlined": true,
                      "path": "/components/sheet",
                      "statusBadges": Array [
                        "javascript",
                        "updated",
                      ],
                      "title": "Sheet",
                    },
                    Object {
                      "componentPath": "components/Select",
                      "icon": "arrow_drop_down_circle",
                      "outlined": true,
                      "path": "/components/select",
                      "statusBadges": Array [
                        "javascript",
                      ],
                      "title": "Select",
                    },
                    Object {
                      "componentPath": "components/Sidebar",
                      "icon": "view_sidebar",
                      "outlined": true,
                      "path": "/components/sidebar",
                      "statusBadges": Array [
                        "javascript",
                      ],
                      "title": "Sidebar",
                    },
                    Object {
                      "componentPath": "components/SkipLink",
                      "icon": "skip_next",
                      "outlined": true,
                      "path": "/components/skip-link",
                      "title": "Skip link",
                    },
                    Object {
                      "componentPath": "components/Slab",
                      "icon": "crop_landscape",
                      "outlined": true,
                      "path": "/components/slab",
                      "title": "Slab",
                    },
                    Object {
                      "componentPath": "components/Status",
                      "icon": "check_circle",
                      "outlined": true,
                      "path": "/components/status",
                      "statusBadges": Array [
                        "updated",
                      ],
                      "title": "Status",
                    },
                    Object {
                      "componentPath": "components/Tables",
                      "icon": "view_list",
                      "outlined": true,
                      "path": "/components/tables",
                      "title": "Tables",
                    },
                    Object {
                      "componentPath": "components/Tabs",
                      "icon": "folder",
                      "outlined": true,
                      "path": "/components/tabs",
                      "statusBadges": Array [
                        "javascript",
                        "updated",
                      ],
                      "title": "Tabs",
                    },
                    Object {
                      "componentPath": "components/CodeTags",
                      "icon": "code",
                      "outlined": true,
                      "path": "/components/tags",
                      "statusBadges": Array [
                        "updated",
                      ],
                      "title": "Tags",
                    },
                    Object {
                      "componentPath": "components/Toast",
                      "icon": "picture_in_picture",
                      "outlined": true,
                      "path": "/components/toast",
                      "statusBadges": Array [
                        "javascript",
                      ],
                      "title": "Toast",
                    },
                    Object {
                      "componentPath": "components/Togglebox",
                      "icon": "toggle_off",
                      "outlined": true,
                      "path": "/components/togglebox",
                      "title": "Togglebox",
                    },
                    Object {
                      "componentPath": "components/Tooltips",
                      "icon": "filter_frames",
                      "outlined": true,
                      "path": "/components/tooltips",
                      "statusBadges": Array [
                        "javascript",
                      ],
                      "title": "Tooltips",
                    },
                    Object {
                      "componentPath": "components/Topbar",
                      "icon": "call_to_action",
                      "outlined": true,
                      "path": "/components/topbar",
                      "statusBadges": Array [
                        "javascript",
                      ],
                      "title": "Topbar",
                    },
                  ],
                  "title": "Components",
                },
                Object {
                  "appFolder": "Patterns",
                  "entryCardText": "Learn about how to we set up different structures",
                  "icon": Object {
                    "name": "dashboard",
                  },
                  "path": "/patterns",
                  "redirect": "/patterns/patterns",
                  "routes": Array [
                    Object {
                      "componentPath": "content/Patterns",
                      "path": "/patterns/patterns",
                      "title": "Overview",
                    },
                    Object {
                      "componentPath": "content/Forms",
                      "icon": "format_align_justify",
                      "outlined": true,
                      "path": "/patterns/forms",
                      "title": "Forms",
                    },
                    Object {
                      "componentPath": "content/Login",
                      "icon": "account_circle",
                      "outlined": true,
                      "path": "/patterns/login",
                      "statusBadges": Array [
                        "new",
                      ],
                      "title": "Login",
                    },
                  ],
                  "title": "Patterns",
                },
                Object {
                  "appFolder": "Utilities",
                  "entryCardText": "See all available CSS classes",
                  "icon": Object {
                    "name": "build",
                  },
                  "path": "/utilities",
                  "redirect": "/utilities/utilities",
                  "routes": Array [
                    Object {
                      "componentPath": "utilities/Utilities",
                      "path": "/utilities/utilities",
                      "title": "Overview",
                    },
                    Object {
                      "componentPath": "utilities/Borders",
                      "path": "/utilities/borders",
                      "text": "Set borders that suit your needs",
                      "title": "Borders",
                    },
                    Object {
                      "componentPath": "utilities/Breakpoints",
                      "path": "/utilities/breakpoints",
                      "text": "See our display breakpoints",
                      "title": "Breakpoints",
                    },
                    Object {
                      "componentPath": "utilities/Colors",
                      "path": "/utilities/colors",
                      "text": "See our color utility classes",
                      "title": "Colors",
                    },
                    Object {
                      "componentPath": "utilities/Display",
                      "path": "/utilities/display",
                      "text": "See our display properties",
                      "title": "Display",
                    },
                    Object {
                      "componentPath": "utilities/Flex",
                      "path": "/utilities/flex",
                      "text": "Layout, alignment and grid",
                      "title": "Flex",
                    },
                    Object {
                      "componentPath": "utilities/Images",
                      "path": "/utilities/images",
                      "text": "Modify how images are displayed",
                      "title": "Images",
                    },
                    Object {
                      "componentPath": "utilities/Sizing",
                      "path": "/utilities/sizing",
                      "text": "Size according to your needs",
                      "title": "Sizing",
                    },
                    Object {
                      "componentPath": "utilities/Text",
                      "path": "/utilities/text",
                      "text": "Control wrapping, weight and more",
                      "title": "Text",
                    },
                    Object {
                      "componentPath": "utilities/Visibility",
                      "path": "/utilities/visibility",
                      "text": "Control visibilities of display",
                      "title": "Visibility",
                    },
                  ],
                  "title": "Utilities",
                },
              ]
            }
          />
        </div>
        <main
          className="doc-view"
          id="doc-view"
        >
          <SearchBox
            className="d-none d-lg-block"
          />
          <Suspense
            fallback={<LoadingComponent />}
          >
            <Switch>
              <Route
                component={
                  Object {
                    "$$typeof": Symbol(react.lazy),
                    "_init": [Function],
                    "_payload": Object {
                      "_result": [Function],
                      "_status": -1,
                    },
                  }
                }
                exact={true}
                path="/"
              />
              <Route
                component={
                  Object {
                    "$$typeof": Symbol(react.lazy),
                    "_init": [Function],
                    "_payload": Object {
                      "_result": [Function],
                      "_status": -1,
                    },
                  }
                }
                path="/get-started"
              />
              <Route
                component={
                  Object {
                    "$$typeof": Symbol(react.lazy),
                    "_init": [Function],
                    "_payload": Object {
                      "_result": [Function],
                      "_status": -1,
                    },
                  }
                }
                path="/components"
              />
              <Route
                component={
                  Object {
                    "$$typeof": Symbol(react.lazy),
                    "_init": [Function],
                    "_payload": Object {
                      "_result": [Function],
                      "_status": -1,
                    },
                  }
                }
                path="/identity"
              />
              <Route
                component={
                  Object {
                    "$$typeof": Symbol(react.lazy),
                    "_init": [Function],
                    "_payload": Object {
                      "_result": [Function],
                      "_status": -1,
                    },
                  }
                }
                path="/examples"
              />
              <Route
                component={
                  Object {
                    "$$typeof": Symbol(react.lazy),
                    "_init": [Function],
                    "_payload": Object {
                      "_result": [Function],
                      "_status": -1,
                    },
                  }
                }
                path="/patterns"
              />
              <Route
                component={
                  Object {
                    "$$typeof": Symbol(react.lazy),
                    "_init": [Function],
                    "_payload": Object {
                      "_result": [Function],
                      "_status": -1,
                    },
                  }
                }
                path="/utilities"
              />
              <Route
                component={
                  Object {
                    "$$typeof": Symbol(react.lazy),
                    "_init": [Function],
                    "_payload": Object {
                      "_result": [Function],
                      "_status": -1,
                    },
                  }
                }
                path="/404"
              />
              <Route
                component={
                  Object {
                    "$$typeof": Symbol(react.lazy),
                    "_init": [Function],
                    "_payload": Object {
                      "_result": [Function],
                      "_status": -1,
                    },
                  }
                }
              />
            </Switch>
          </Suspense>
        </main>
      </div>
    </div>
  </withRouter(ScrollToTop)>
</Router>
`;
